src/browser-history.ts

```diff
@@ -39,7 +37,7 @@ export class BrowserHistory extends History {
     super();
 
     this._isActive = false;
-    this._checkUrlCallback = this._checkUrl.bind(this);
+    this._checkUrl = this._checkUrl.bind(this);
```

Comment:
Any reason why this change was made?

Reply:
The old way is less bundle size friendly

Reply:
Doesn't this change the behavior though, since the same property is representing both a bound and unbound function?

Reply:
Yes, it changes instance behavior. Anyone who wants prototype fn can still get it via `BrowserHistory.prototype._checkUrl`.